src/docfx/build/page/OutputModel.cs

```diff
@@ -49,7 +53,16 @@ public class OutputModel : InputMetadata
 
         public string SiteName { get; set; }
 
-        public string Conceptual => Content as string;
+        public string DepotName { get; set; }
+
+        [JsonProperty("_path")]
```

Comment:
`_path` can can be removed

Reply:
_path is used to calculate canonical url in template:

![image](https://user-images.githubusercontent.com/19990166/57291145-71315b00-70f1-11e9-9d9a-55cd12a07cb5.png)

Reply:
ok, we then need to update template code after next weeks meeting, because `_versionPath` is also missing.